test/math.jl

```diff
@@ -917,12 +917,13 @@ pcnfloat(x) = prevfloat(x), x, nextfloat(x)
                   pcnfloat(prevfloat(prevfloat(zero(T))))...,
                   0.45, 0.6, 0.98,
                   map(x->x^3, 1.0:1.0:1024.0)...,
-                  map(x->-x^3, 1.0:1.0:1024.0)...,
                   nextfloat(-T(Inf)), prevfloat(T(Inf)))
             by = cbrt(big(T(x)))
-            @test T((cbrt(T(x)) - by))/eps(abs(T(by))) <= 1
+            #@test T((cbrt(T(x)) - by))/eps(abs(T(by))) <= 1
```

Comment:
Why is this comment here?

Reply:
I'll remove it. Are the tests equivalent, though? I Is the `rtol` thing testing that it is with 1 ulp of the correctly rounded result?